opencv.html.markdown

```diff
@@ -0,0 +1,141 @@
+---
+category: tool
+tool: OpenCV
+filename: learnopencv.py
+contributors:
+    - ["Yogesh Ojha", "http://github.com/yogeshojha"]
+---
+### Opencv
+
+OpenCV (Open Source Computer Vision) is a library of programming functions mainly aimed at real-time computer vision. 
+Originally developed by Intel, it was later supported by Willow Garage then Itseez (which was later acquired by Intel). 
+Opencv currently supports wide variety of languages like, C++, Python, Java etc
+
+#### Installation
+Please refer to these articles for installation of OpenCV on your computer.
+* Windows Installation Instructions: [https://opencv-python-tutroals.readthedocs.io/en/latest/py_tutorials/py_setup/py_setup_in_windows/py_setup_in_windows.html#install-opencv-python-in-windows]
```

Comment:
Please add a newline before this line ie.

```
Please refer to these articles for installation of OpenCV on your computer.

* Windows Installation Instructions
```

Reply:
Also, insert a `()` at the end of the link ie.

```
* Windows Installation Instructions: [https://opencv-python-tutroals.readthedocs.io/en/latest/py_tutorials/py_setup/py_setup_in_windows/py_setup_in_windows.html#install-opencv-python-in-windows]()
```

This has to be done for **all** links present in this list.